cmake/cpp_bindgen.cmake.in

```diff
@@ -35,7 +35,7 @@ set(__CPP_BINDGEN_SOURCE_DIR @__CPP_BINDGEN_SOURCE_DIR@)
 set(__CPP_BINDGEN_INCLUDE_DIR @__CPP_BINDGEN_INCLUDE_DIR@)
```

Comment:
above, I think it should be
```
include_guard(GLOBAL)
```
because functions are defined globally, right?

Reply:
Thanks, I wasn't aware of different scoping for the include_guard